<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbl_n">
          <property name="font">
           <font>
            <family>Courier New</family>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>n</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_n">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_l">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>l</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_l"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_m">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>m</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_m"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QComboBox" name="cmbSysCoord">
        <item>
         <property name="text">
          <string>cartesian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>spherical</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="lbl_x_r_2">
          <property name="font">
           <font>
            <family>Courier New</family>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>lower limits</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lblMin_x_r">
              <property name="font">
               <font>
                <family>Courier New</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>x</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMin_x_r">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>-1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lblMin_y_theta">
              <property name="font">
               <font>
                <family>Courier New</family>
                <pointsize>11</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMin_y_theta">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>-1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="lblMin_z_phi">
              <property name="font">
               <font>
                <family>Courier New</family>
                <pointsize>11</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMin_z_phi">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>-1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="lbl_x_r_3">
          <property name="font">
           <font>
            <family>Courier New</family>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>upper limits</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="lblMax_x_r">
              <property name="font">
               <font>
                <family>Courier New</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>x</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMax_x_r">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="lblMax_y_theta">
              <property name="font">
               <font>
                <family>Courier New</family>
                <pointsize>11</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMax_y_theta">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="lblMax_z_phi">
              <property name="font">
               <font>
                <family>Courier New</family>
                <pointsize>11</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinMax_z_phi">
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnBuildPlot">
        <property name="text">
         <string>Build Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>214</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
